lang_java/parsing/parser_java.mly

```diff
@@ -883,13 +883,13 @@ statement_without_trailing_substatement:
 block: "{" block_statement* "}"  { Block ($1, List.flatten $2, $3) }
 
 block_statement:
- | local_variable_declaration_statement  { $1 }
```

Comment:
We probably can return a single statement now, instead of returning a list with a single element each time.

Reply:
I'll do a second PR to refactor